R/benchmark_shinytest2.R

```diff
@@ -62,7 +63,7 @@ benchmark_shinytest2 <- function(
         restore_env(branch = current_branch, renv_prompt = renv_prompt)
 
       # Cleaning the temporary directory
-      unlink(x = file.path(project_path, "tests"), recursive = TRUE)
+      unlink(fs::path(project_path, "tests"))
```

Comment:
fs::file_delete maybe? to keep the pattern

Reply:
I think this one will have to stick. When using `fs::file_delete` it creates a problem with CI/CD as it cannot delete a file that is being used (EBUSY) error.

I couldn't figure out what process was using it and had to revert to `unlink`. I'll check one more time and place a comment indicating why it doesn't use `fs`